modules/claims_api/config/schemas/526.json

```diff
@@ -171,12 +171,18 @@
           "description": "A Change of Address Object being submitted.",
           "type": "object",
           "additionalProperties": false,
+          "required": ["country", "type", "addressLine1", "addressChangeType", "beginningDate"],
           "properties": {
             "beginningDate": {
               "description": "Date in YYYY-MM-DD the Veteran changed address.",
               "type": "string",
               "example": "2018-06-04"
             },
+            "endingDate": {
+              "description": "Date in YYYY-MM-DD the changed address expires, if change is temporary.",
+              "type": "string",
+              "example": "2018-06-04"
+            },
```

Comment:
Should we apply a date pattern to this, similar to [here](https://github.com/department-of-veterans-affairs/vets-api/blob/2fe6a2ce2355cd5d05d2ec5d04c8698c76f7f9dc/modules/claims_api/config/schemas/526.json#L464)?

Reply:
done. also removed all the XXXX-XX-XX from other patterns
https://github.com/department-of-veterans-affairs/vets-api/pull/7711/commits/0fe8d82fb309953deeb237eb0bc08e1e986c157f